src/log_batch.rs

```diff
@@ -616,28 +616,68 @@ pub fn decompress(buf: &[u8]) -> Vec<u8> {
 #[cfg(test)]
 mod tests {
     use super::*;
+
+    use protobuf::parse_from_bytes;
     use raft::eraftpb::Entry;
 
+    fn entries(first_index: u64, len: usize, data: Option<Vec<u8>>) -> Vec<Entry> {
```

Comment:
similar to `generate_entry_indexes`? How about sharing it in a `test_util` file?